about/introduction.rst

```diff
@@ -1,14 +1,17 @@
 Introduction
 ============
 
-Our goal is to create a copylefted libre mobile operating system. One you can use, study, change and share; with all.
+Our goal is to create a copylefted libre mobile operating system. One you can use, study, change and share; with all of the code released under the GNU General Public license, V It's completely free, and anyone can inspect or modify the source code of any program that is included.
```

Comment:
The sentence was a bit clunky with respect to punctuation, but now it's really unwieldy.

What's that V for?

Reply:
actually are we sure that *all* of the code is released under the GPL? Also, the L in license should be capitalized

Reply:
Remove this part than " V It's completely free"

Reply:
> The sentence was a bit clunky with respect to punctuation, but now it's really unwieldy.

Ok, so to make this more concrete, I'd suggest to replace the ; with a . and start a new sentence. I think that would do

However, the more I think about the GPL, the more I'm doubting. There are most definitely parts that are not GPL, e.g., X or Wayland